.github/workflows/molecule.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: Molecule Tests
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  molecule:
+    name: molecule
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        scenario:
+          - default
+          - create-destroy
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: pip3 install molecule[docker] ansible-core
+
+      - name: Run molecule
+        run: molecule test -s ${{ matrix.scenario }}
+        env:
+          PY_COLORS: '1'
+          ANSIBLE_FORCE_COLOR: '1'
```

Comment:
```suggestion
          ANSIBLE_FORCE_COLOR: '1'
          ANSIBLE_STDOUT_CALLBACK: 'yaml'
```

maybe?

Reply:
Yeah seems reasonable, what does that look like? I just tried `debug` last night and that was ok-ish.

Reply:
```
TASK [galaxyproject.tiaas2 : Install base dependencies] **********************************************************************************************************************
fatal: [gat-0.eu.training.galaxyproject.eu]: FAILED! => changed=false
  cmd:
  - /srv/tiaas/venv/bin/pip3
  - install
  - --index-url
  - https://wheels.galaxyproject.org/simple/
  - --extra-index-url
  - https://pypi.python.org/simple
  - -r
  - /srv/tiaas/code/requirements.txt
  msg: |-
    stdout: Looking in indexes: https://wheels.galaxyproject.org/simple/, https://pypi.python.org/simple
    Collecting dateparser==0.7.2
      Using cached dateparser-0.7.2-py2.py3-none-any.whl (352 kB)
    Collecting Django==3.1.14
      Using cached Django-3.1.14-py3-none-any.whl (7.8 MB)
    Collecting django-bootstrap3==21.2
      Using cached django_bootstrap3-21.2-py3-none-any.whl (213 kB)
    Collecting django-countries==5.5
      Using cached django_countries-5.5-py2.py3-none-any.whl (764 kB)
    Collecting django-prometheus==2.2.0
      Using cached django_prometheus-2.2.0-py2.py3-none-any.whl (29 kB)
    Collecting gunicorn==20.1.0
      Using cached gunicorn-20.1.0-py3-none-any.whl (79 kB)
    Collecting psycopg2-binary==2.8.4
      Using cached psycopg2-binary-2.8.4.tar.gz (378 kB)
      Preparing metadata (setup.py): started
      Preparing metadata (setup.py): finished with status 'done'
    Collecting pycryptodome==3.15.0
      Using cached pycryptodome-3.15.0-cp35-abi3-manylinux2010_x86_64.whl (2.3 MB)
    Collecting factory_boy
```

Reply:
it looks realy nice.

Reply:
Oh wow, that is nice - feel free to add in the future.